src/ExtractGQL.ts

```diff
@@ -109,13 +109,13 @@ export class ExtractGQL {
     inputFilePath,
     outputFilePath = 'extracted_queries.json',
     queryTransformers = [],
-    extension = 'graphql',
+    extension = '',
     inJsCode = false,
   }: ExtractGQLOptions) {
     this.inputFilePath = inputFilePath;
     this.outputFilePath = outputFilePath;
     this.queryTransformers = queryTransformers;
-    this.extension = extension;
+    this.extension = this.extension.concat(extension.split(',').map(e => e.trim()))
```

Comment:
Instead of doing this, we should just have a `this.extensions` that we can pass in.

Reply:
Can you clarify this? If it's coming from the CLI how else do you propose I pass in an array of things? I could just let the variable be a string, and split it later?